contracts/interfaces/IPausableAdmin.sol

```diff
@@ -18,3 +18,20 @@ interface IPausableAdmin {
 
     function renouncePauseAdmin() external;
 }
+
+interface IPausableAdminUpgradeable {
+    event PauseAdminAdded(address newAdmin);
+    event PauseAdminRemoved(address sender, address removedAdmin);
+
+    function getPauseAdminAt(uint256 _index) external view returns (address);
+
+    function isPauseAdmin(address _user) external view returns (bool);
+
+    function getNumberOfPauseAdmin() external view returns (uint256);
+
+    function addPauseAdmin(address _newAdmin) external;
+
+    function removePauseAdmin(address _admin) external;
+
+    function renouncePauseAdmin() external;
+}
```

Comment:
Please put this in a separate file, because when you verify the contract it will increase the length of the contract for no reason